dbcrossbar/src/clouds/gcloud/client.rs

```diff
@@ -222,6 +222,68 @@ impl Client {
         }
     }
 
+    /// Make an HTTP POST request and return the response.
+    async fn post_helper<Body>(
```

Comment:
```suggestion
    /// Make an HTTP POST request and return the response.
    ///
    /// This may POST the request multiple times, which may cause an action to be
    /// performed multiple times. The caller is responsible for ensuring that the action
    /// is idempotent (perhaps by using `IF NOT EXISTS` for an SQL operation), and
    /// being ready to handle the case where the underlying action succeeds, but the
    /// server still returns an error.
    async fn post_with_retry_helper<Body>(
```

POST is not normally idempotent, so this can't be a generic `post_helper`. The caller